mne_bids/tests/test_write.py

```diff
@@ -491,6 +545,12 @@ def test_bdf(_bids_validate):
     with pytest.raises(AssertionError, match='cropped'):
         write_raw_bids(raw, bids_basename, output_path)
 
+    # test anonymize
+    output_path = _TempDir()
+    raw = mne.io.read_raw_bdf(raw_fname)
+    with pytest.warns(UserWarning, match='Converting to FIF to anonymize'):
```

Comment:
hmm ... but it's EEG data -- why should it convert to FIF? It's not BIDS compatible any more then

Reply:
It won't be BIDS compatible until it's converted back by #237 see above I don't know if there's a better way to do that

Reply:
okay let's merge #237 first then! Address my tiny comment there and I will merge.